airflow/models.py

```diff
@@ -56,8 +56,8 @@
 
 from sqlalchemy import (
     Column, Integer, String, DateTime, Text, Boolean, ForeignKey, PickleType,
-    Index, Float, LargeBinary, UniqueConstraint)
-from sqlalchemy import func, or_, and_, true as sqltrue
+    Index, Float, LargeBinary, UniqueConstraint, ForeignKeyConstraint)
+from sqlalchemy import func, or_, and_, true as sqltrue, asc
```

Comment:
how does line work(`import func, or_, and_, true as sqltrue, asc`) ? I wonder whether you want to do `from sqlalchemy import asc` instead?

Reply:
There are already two `from sqlalchemy import ...` statements: the first imports types, the second imports (SQL) expressions.
I can introduce a third one.
Or combine all into one like this (lexicographically sorted):
```
from sqlalchemy import (
    Boolean, Column, DateTime, Float, ForeignKey, ForeignKeyConstraint, Index,
    Integer, LargeBinary, PickleType, String, Text, UniqueConstraint, 
    and_, asc, func, or_, true as sqltrue
)
```

Reply:
I changed it like this, let me know what you think.